src/z3c/rml/pagetemplate.txt

```diff
@@ -11,41 +11,44 @@ them in the PDF.
 
```

Comment:
Can we move it to `./README.pagetemplate.txt` or smth? It can be still useful, even if it doesn't run as doctest.